src/java/org/apache/cassandra/db/virtual/AccordVirtualTables.java

```diff
@@ -18,57 +18,162 @@
 
 package org.apache.cassandra.db.virtual;
 
-import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
 
-import com.google.common.annotations.VisibleForTesting;
-
+import accord.local.CommandStores;
+import accord.primitives.TxnId;
+import accord.utils.async.AsyncChains;
 import org.apache.cassandra.config.DatabaseDescriptor;
 import org.apache.cassandra.cql3.statements.schema.CreateTableStatement;
+import org.apache.cassandra.dht.Range;
+import org.apache.cassandra.dht.Token;
 import org.apache.cassandra.schema.TableMetadata;
+import org.apache.cassandra.service.accord.AccordCommandStore;
 import org.apache.cassandra.service.accord.AccordService;
-import org.apache.cassandra.service.accord.IAccordService;
+import org.apache.cassandra.service.accord.AccordStateCache;
+import org.apache.cassandra.service.consensus.migration.ConsensusMigrationState;
+import org.apache.cassandra.service.consensus.migration.TableMigrationState;
+import org.apache.cassandra.tcm.ClusterMetadata;
+import org.apache.cassandra.utils.Clock;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
 
 public class AccordVirtualTables
 {
-    private AccordVirtualTables()
-    {
-
-    }
+    private AccordVirtualTables() {}
 
     public static Collection<VirtualTable> getAll(String keyspace)
     {
         if (!DatabaseDescriptor.getAccordTransactionsEnabled())
             return Collections.emptyList();
 
-        return Arrays.asList(
-        new Epoch(keyspace)
+        return List.of(
+            new CommandStoreCache(keyspace),
+            new MigrationStates(keyspace),
+            new CoordinationStatus(keyspace)
         );
     }
 
-    @VisibleForTesting
-    public static final class Epoch extends AbstractVirtualTable
+    public static final class CommandStoreCache extends AbstractVirtualTable
     {
+        private CommandStoreCache(String keyspace)
```

Comment:
it would be useful to include instance as well, either as its own table or a column here... having a low hit rate doesn't let you know what's going on, need to see which instances are contributing to it.

Reply:
Is there even any reason to populate this w/ the global `Stats`? Would it make more sense to expose the `Stats` from `commandCache`, `timestampsForKeyCache`, and `commandsForKeyCache` by store ID instead?